src/Tribe/Main.php

```diff
@@ -351,6 +354,9 @@ public function bind_implementations() {
 		// Attendee Registration Page
 		tribe_register_provider( 'Tribe__Tickets__Attendee_Registration__Service_Provider' );
 
+		// Event Tickets Provider to manage Events
```

Comment:
:no_entry_sign: Inline comments must end in full-stops, exclamation marks, or question marks

<sub>hash: dac5a3da614956a99f2f262e8a4587ec</sub>